Makefile

```diff
@@ -72,6 +72,11 @@ menuconfig: Kconfig
 	MENUCONFIG_STYLE=aquatic \
 	$(KCONFIG_MCONF) $<
 
+# Generate config for the current architecture (only)
+# This is done to avoid a dependency on KCONFIG tools
+defconfig:
```

Comment:
What does the name `defconfig` mean? If it stands for something like 'default config,' wouldn't `config.mk` already be the default? I believe the name should be something that users searching for the feature added by this PR can easily find.

Reply:
It's a kconfig thing, I think it stands for default config for this specific architecture or some-such?
It's unrelated to the "default config" (all options off / empty config file), in that it does have config.

The way it is created is that you select the options you want, and then run "savedefconfig" to defconfig.
The normal .config file that is generated has **all** available options, including ones you didn't pick.

Reply:
It is useful when you are building something linux or busybox, or an embedded system like buildroot.

It is totally overkill for something with a half dozen options (like Lima) - unless you are already used to it :-)

Reply:
The reason for doing a single script instead of multiple files, is that there would be duplication of other config.